hikari/impl/rest.py

```diff
@@ -3847,9 +3865,35 @@ async def create_autocomplete_response(
         body.put("data", data)
         await self._request(route, json=body, no_auth=True)
 
+    async def create_modal_response(
+        self,
+        interaction: snowflakes.SnowflakeishOr[base_interactions.PartialInteraction],
+        token: str,
+        *,
+        title: str,
+        custom_id: str,
+        components: typing.Sequence[special_endpoints.ComponentBuilder],
```

Comment:
There will always have to be an action row here, no? So this will just always be a list of one item?

Reply:
I though it'd be a list of action rows, each with one component as of now.

Reply:
You are correct, forgot they all have to be wrapped around different action rows